### PR TITLE
[Outlook] (activation rules) - Update info about ItemHasRegularExpressionMatch rule

### PR DESCRIPTION
**Pull request by kbrandl**
Fixes issue [370](https://github.com/OfficeDev/outlook-dev-docs/issues/370) in **OfficeDev/outlook-dev-docs**.

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[d8c0c20](https://github.com/OfficeDev/office-js-docs-pr/pull/650/commits/d8c0c206178dda6232100bca9c9164af853f2d9e)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[d8c0c20](https://github.com/OfficeDev/office-js-docs-pr/pull/650/commits/d8c0c206178dda6232100bca9c9164af853f2d9e)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[d8c0c20](https://github.com/OfficeDev/office-js-docs-pr/pull/650/commits/d8c0c206178dda6232100bca9c9164af853f2d9e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/rule.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-outlook-issue-370/docs/reference/manifest/rule.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812272046250982-650%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=psRmGxbos7uMG6nMYopvlp%2BLsm3P82KYpiGDVltvavk%3D&st=2018-12-27T20%3A42%3A40Z&se=2019-01-27T20%3A47%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/rule.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ Defines a rule that evaluates to true if a match for the specified regular expre
...
 | **RegExValue** | Yes | Specifies the regular expression that will be evaluated to determine whether the mail add-in should be shown. |
-| **PropertyName** | Yes | Specifies the name of the property that the regular expression will be evaluated against. Can be one of the following: `Subject`, `BodyAsPlaintext`, `BodyAsHTML`, or `SenderSMTPAddress`. |
-| **IgnoreCase** | No | Specifies to ignore the case when executing the regular expression. |
+| **PropertyName** | Yes | Specifies the name of the property that the regular expression will be evaluated against. Can be one of the following: `Subject`, `BodyAsPlaintext`, `BodyAsHTML`, or `SenderSMTPAddress`.<br/><br/>If you specify `BodyAsHTML`, Outlook applies the regular expression only if the item body is HTML, and otherwise Outlook returns no matches for that regular expression.<br/><br/>If you specify  `BodyAsPlaintext`, Outlook always applies the regular expression on the item body.<br/><br/>**Note:** You must set the **PropertyName** attribute to `BodyAsPlaintext` if you specify the **Highlight** attribute for the **Rule** element.|
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/reference/manifest/rule.md]: Will do.

[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/reference/manifest/rule.md]: Will do.

**Pull request merged**